playbooks/group_vars/webportals.yml

```diff
@@ -113,16 +113,24 @@ webportal_allowance:
 webportal_server_config_defaults:
   domain_name: "{{ ansible_host }}"
   portal_name: "{{ ansible_host }}"
-  portal_modules: ""
+  portal_modules: "m"
```

Comment:
do we want to also include accounts by default or we want to default to as minimal setup as possible? including just mongo sounds redundant

Reply:
I went with the minimum, so no accounts. Since our docker compose files set the mongo env vars for skyd, skyd won't start without mongo.  So even though mongo isn't required for a bare bones skyd instance, because ansible is using our docker compose files, mongo becomes required. 

tl;dr minimum set up requires mongo.

Reply:
> Since our docker compose files set the mongo env vars for skyd, skyd won't start without mongo

This is not correct any more, I fixed that couple of days ago when I tried starting without mongo and it worked fine :) https://github.com/SkynetLabs/skynet-webportal/commit/dba33f833569968b8656ca8b53a3cec5d7090caf

Reply:
Nice 👍 

I think we can leave this for now. If we want to change it in the future we can.